Yehor/reactNetflix/src/pages/detailedPage/DetailedPage.tsx

```diff
@@ -7,39 +7,75 @@ import SortByGenrePanel from "../../components/sortBygenre-panel";
 import { RouteComponentProps } from "react-router-dom";
 import IMovies from "../../interfaces/IMovies";
 import IMovie from "../../interfaces/IMovie";
+import {
+  CurrentMovieLoad,
+  CurrentMovieReceived,
+} from "../../redux/actions/movieActions";
+import { connect, ConnectedProps } from "react-redux";
 
-type DetailedPageProps = IMovies & RouteComponentProps<{ id: string }>;
-
-function DetailedPage({ movies, match }: DetailedPageProps) {
-  const currentMovie = movies.find(
-    (movie) => String(movie.id) === match.params.id
-  );
-  return (
-    <div className="allpageWrapper">
-      <div className="wrapper">
-        <div className="blur">
-          <div className="top-container">
-            <Header />
-            {currentMovie ? (
-              <DetailedMovie movie={currentMovie} />
-            ) : (
-              <div>There is no movie</div>
-            )}
+interface IRootState {
```

Comment:
IRootState is a reusable interface, so move move it to the file with store or in some separate interfaces file